habitat-lab/habitat/robots/mobile_manipulator.py

```diff
@@ -26,8 +26,9 @@ class RobotCameraParams:
     """
 
     attached_link_id: int
-    cam_offset_pos: mn.Vector3
-    cam_look_at_pos: mn.Vector3
+    cam_offset_pos: mn.Vector3 = mn.Vector3(0, 0, 0)
+    cam_look_at_pos: mn.Vector3 = mn.Vector3(0, 0, 0)
+    cam_orientation: mn.Vector3 = mn.Vector3(0, 0, 0)
```

Comment:
```suggestion
    cam_offset_pos: mn.Vector3 = mn.Vector3.zero_init()
    cam_look_at_pos: mn.Vector3 = mn.Vector3.zero_init()
    cam_orientation: mn.Vector3 = mn.Vector3.zero_init()
```
I believe this is marginally faster.

Reply:
Thank you! :)